src/ert/scheduler/job.py

```diff
@@ -164,7 +163,7 @@ async def run(self, sem: asyncio.BoundedSemaphore, max_submit: int = 1) -> None:
                 break
 
             if self.returncode.result() == 0:
-                if self._scheduler.wait_for_checksum():
+                if self._scheduler._manifest_queue:
```

Comment:
at first sight, this looks like there can be a bug introduced when the manifest queue is not yet populated, then it will skip it?

Reply:
I think I misread the code. Should we write `if self._scheduler._manifest_queue is not None` to be more explicit? (I thought we were testing whether there is anything in the queue)